h2o-core/src/main/java/water/util/MathUtils.java

```diff
@@ -773,4 +773,14 @@ public static double logloss(double err) {
     }
     return transposed;
   }
+
+  /**
+   * Compute harmonic number estimated by natural logarithm
+   * Constant here: Euler–Mascheroni constant (also called Euler's constant)
```

Comment:
instead of a comment I would put the constant into a public static final field with an explanatory name

Reply:
Implemented in PR: https://github.com/h2oai/h2o-3/pull/4907